mona_openai/endpoints/chat_completion.py

```diff
@@ -164,6 +173,7 @@ def _get_full_textual_analysis(
                 if analyzer.get_word_count() > 0
                 else 0.0
                 for analyzer in answers_textual_analyzers
+                if not analyzer.is_none_analyzer()
```

Comment:
Can we iterate combined_messages instead of answers_textual_analyzers and save this line? Or am I understanding create_combined_analyzer wrong?

Reply:
I can't see how this will help us.